ansible/environments/docker-machine/group_vars/all

```diff
@@ -0,0 +1,40 @@
+whisk_version_name: mac
```

Comment:
is this the mac file moves to docker-machine (ie just renamed)? I'm surprised git didn't pick it up as a move.
edit: ok i see why.